contracts/interfaces/IUniswapV3Staker.sol

```diff
@@ -37,9 +37,9 @@ interface IUniswapV3Staker {
         address pool;
         address rewardToken;
         uint128 totalReward;
-        uint32 claimDeadline;
-        uint32 endTime;
         uint32 startTime;
+        uint32 endTime;
```

Comment:
Good catch on these. I had sorted all the struct members alphabetically (so data type a-z, then name a-z), but it makes more sense to have the timestamps ordered this way.